### DISCUSSION TITLE
Error in building the 'hello world' example.

### DISCUSSION THREAD

**Discussion by yangzxatimech** [AMReX-Codes/amrex#2799] [Q&A]
When I tried to build the 'helloworld' example, I met the following error:

-------------------------------------------------------------------------------------------------------
fatal: Not a git repository (or any parent up to mount point /public3)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Loading ../../../../amrex/Tools/GNUMake/comps/intel.mak...
Loading ../../../../amrex/Tools/GNUMake/sites/Make.unknown...
../../../../amrex/Tools/GNUMake/Make.rules:495: tmp_build_dir/d/3d.intel.DEBUG.EXE/f90.depends: No such file or directory
Generating AMReX_Version.H ...
/bin/sh: ../../../../amrex/Tools/libamrex/mkversionheader.py: Permission denied
make: *** [tmp_build_dir/s/3d.intel.DEBUG.EXE/AMReX_Version.H] Error 126
---------------------------------------------------------------------------------------------------------

Is there any idea on how to fix the issue?

**Comment by etpalmer63** [comment]
Hi Yang,

Judging by the error message it seems there could be some filesystem and execution privileges that you need but do not have. Have you had other issues with git repositories? I think it would be worth it to try `git clone https://github.com/AMReX-Codes/amrex.git` to download the AMReX repo. Then navigate to the directory you just cloned, `/amrex`, and type `git status` and post the result here.

Also, could you tell me a little more about the the steps and commands you made that gave you these messages? With this info I can probably revise the directions in the helloworld example  to be more comprehensive. 

Thanks for your question,

Erik

**Comment by WeiqunZhang** [comment]
`fatal: Not a git repository` is a fatal error to git, but not to amrex.

`/f90.depends: No such file or directory` is fine.

`/bin/sh: ../../../../amrex/Tools/libamrex/mkversionheader.py: Permission denied`.  As @etpalmer63 said, the file does not have the correct permission to be executed.
